### PR TITLE
Fixed megamenu link to go to edit profile flow

### PR DESCRIPTION
**Pull request by alalonde**
Also, closes https://github.com/MetaFam/TheGame/issues/830

### CODE REVIEW

**Comment by mquellhorst** [comment]
I noted this one before approving the merge and proposed to add the edit link to the menu but this seems to be intentional left out on @davort request communicated by @phys3

**Comment by davort** [comment]
You are correct @mquellhorst - the idea is that it's very easy to jump into "Edit" mode once you're looking at your own profile page. At the same time, "View profile" was always missing from the menu, and my bet is that people go look at their profiles as equally, if not more often than editing them 😊 Design decision.

**Comment by mquellhorst** [comment]
This PR is up already so rather have @alalonde change this one.
@davort what you say about just having both the edit and view profile links in the dropdown for now?
This way we can prob get it in the merge with master now

File: packages/web/components/MegaMenu.tsx [hunk 1]
```diff
@@ -311,10 +312,10 @@ const PlayerStats: React.FC<PlayerStatsProps> = ({ player }) => {
         <MenuList mt="8px" color="black" fontFamily="exo2">
           <MetaLink
             color="black"
-            href={`/player/${getPlayerName(player)}`}
+            href="/profile/setup/username"
             _hover={{ textDecoration: 'none' }}
           >
-            <MenuItem>View Profile</MenuItem>
```
Comment (mquellhorst) [inline packages/web/components/MegaMenu.tsx:317]: Discussed with @davort 
Keep both links for now please :)

**Review by mquellhorst — CHANGES_REQUESTED** [review]

**Comment by alalonde** [comment]
OK, I will add a "view profile"  link to the menu and "edit profile" to your own player page.

**Comment by alalonde** [comment]
@mquellhorst Added these both, please re-review

**Review by mquellhorst — APPROVED** [review]

**Pull request merged**